src/Containers/Chart.tsx

```diff
@@ -0,0 +1,299 @@
+                        <ExportIcon />
+                    </Icon>
+                    <strong>Export</strong>
+                </IconControl>
+                <IconControl>
+                    <Icon>
+                        <PrintIcon />
+                    </Icon>
+                    <strong>Print</strong>
+                </IconControl>
+            </Icons>
+        </Header>
+        <LineChart width={460} height={160} data={data}>
+            <XAxis dataKey="label" tickLine={false} />
+            <YAxis axisLine={false} tickLine={false} />
+            <Tooltip
+                cursor={false}
+                content={
+                    <CustomTooltip color={color} active={false} payload={[]} />
+                }
+            />
+            <CartesianGrid vertical={false} />
+            <Line type="linear" dataKey="value" stroke={color} />
+        </LineChart>
+    </ChartBox>
+);
+
+interface ChartData {
+    label: string;
+    value: number;
+}
+
+interface ChartBoxProps
+    extends MainInterface,
+        ResponsiveInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    color: string;
+    revenue: string | number;
+    data: Array<ChartData>;
+}
+
+const ChartBox = styled.div<ChartBoxProps>`
+    // Theme Stuff
+    ${({ theme, ...props }): string => `
+    border-radius: ${theme.dimensions.radius};
+    font-family: ${theme.font.family};
+    color: ${theme.colors.text};
+    ${Main({
+        ...props,
+    })}
+    `}
+    // Base Styles
+    ${Responsive}
+    ${flex('column')}
+    width:fit-content;
+    overflow: hidden;
+`;
+
+const Header = styled.div`
+    ${flex('space-between')}
+    padding-left:7%;
+    margin-bottom: 5%;
+`;
+
+const HeaderLeftWrapper = styled.div`
+    ${flex('center')}
+`;
+
+const HeaderLeft = styled.div`
+    flex: 1;
+    ${flex('column')}
+`;
+
+interface HeaderLeftTitleProps extends React.HTMLAttributes<HTMLDivElement> {
+    color: string;
+}
+
+const HeaderLeftTitle = styled.div<HeaderLeftTitleProps>`
+    ${({ color }): string => `
+color:${color};
+`}
+`;
+
+const HeaderLeftSubtitle = styled.div`
+    ${({ theme }): string => `
+font-size:${theme.font.size.small};
+color:${theme.colors.status.complete};
+`}
+`;
+
+const IconControl = styled.div`
+    ${flex()}
+    margin-right:10%;
+`;
+
+const Icon = styled.div`
+    margin-right: 4%;
+`;
+
+const Icons = styled.div`
+    display: inline-flex;
+    flex: 1;
+    position: relative;
+    left: 17%;
+`;
+
+interface CustomTooltipProps extends CustomTooltipBoxProps {
+    color: string;
+}
+
+const CustomTooltip: React.FC<CustomTooltipProps> = ({
+    active,
+    payload,
+    ...props
+}): React.ReactElement => {
+    const getFormattedHoverValue = () => payload[0].value.toFixed(2);
+
+    if (active) {
+        return (
+            <CustomTooltipBox active={active} payload={payload} {...props}>
+                <strong>
+                    $
+                    <Big>{`${getFormattedHoverValue()}`}</Big>
```

Comment:
```suggestion
                    <Big>${getFormattedHoverValue()}</Big>
```
Can't you just do this instead? If not, just cast the toFixed() with .toString()

Reply:
I did that way because I needed the dollar sign be smaller than the number